aries_cloudagent/protocols/present_proof/v1_0/handlers/presentation_problem_report_handler.py

```diff
@@ -30,16 +30,14 @@ async def handle(self, context: RequestContext, responder: BaseResponder):
             raise HandlerException(
                 "Connection used for presentation problem report not ready"
             )
-        elif not context.connection_record:
-            raise HandlerException(
-                "Connectionless not supported for presentation problem report"
-            )
 
         presentation_manager = PresentationManager(context.profile)
         try:
             await presentation_manager.receive_problem_report(
                 context.message,
-                context.connection_record.connection_id,
+                context.connection_record.connection_id
+                if context.connection_record is not None
+                else None,
```

Comment:
Support passing None to receive_problem_report as the connection_record will not exist if it's connectionless